### PR TITLE
Simplify proposals process

### PR DESCRIPTION
**Pull request by chrisdoherty4**
Change the proposals process to be more light weight.

### CODE REVIEW

**Comment by chrisdoherty4** [comment]
@jacobweinstock @displague

**Comment by displague** [comment]
Hmmm. Moving everything to discussions? Can we create a discussion template for a proposal? 

One potential problem is that discussions are only editable by the original author. What does it look like to accept a proposal from a discussion and then revisit that proposal later if the author is no longer active in the community? Would we create a new discussion for that proposal? Pin a comment directing visitors of the old to the new (if that's possible). Use tags to identify active proposals?  Or is this the role of project docs?  Proposals are then  ephemeral with their artifacts being code change and documentation?

**Comment by chrisdoherty4** [comment]
@displague If discussions aren't flexible enough we can just use issues. I don't have a ton of experience with discussions so I wasn't sure.

Regarding templates, I don't think that's a strong motivator either way.

What do you think?

File: README.md [hunk 1]
```diff
@@ -1,9 +1,43 @@
...
-understand how far a current proposal is far from being published, aborted or
-resolved (committed).
+The proposals repository exists to keep contributors and new people to the project up to speed
+with project activity. Given our small community of contributors, our proposals process is
+deliberately light weight. A light weight process lets us move quickly while still highlighting
+key details for the community.
 
-I hope to see one from you and feel free to collaborate with us leaving your
-comment on currently open proposals.
+To create a proposal, raise a discussion on this repository with pieces from the template below
```
Comment (displague) [inline README.md]: We can link this to the create discussion page

File: README.md [hunk 2]
```diff
@@ -1,9 +1,43 @@
...
+boundaries require additional information to help readers understand what contracts look like.
+For example, what does the contract between 2 services look like? What technology/protocol will 
+they use?
+
+## Rationale
+
+Summarise alternatives that were considered and elaborate on why they are unsatisfactory by
+comparison. There is significant value in portraying options to readers so they can weigh
+up pros and cons. It can be difficul to generate alternatives but that shouldn't be used as a
+reason to avoid it so do think carefully before removing this section.
```
Comment (displague) [inline README.md:43]: "Alternatives considered" is a nice extra to have

This section could link to similar systems in other projects, alternative prs/proposals in Tinkerbell, and discuss decisions made in the course of drafting the proposal.

**Review by jacobweinstock — APPROVED** [review]

**Review by displague — APPROVED** [review]
I had this review stuck in pending for some time. Approving as-is. We can revisit later if there is any interest.

**Comment by displague** [comment]
@mergifyio help

**Comment by displague** [comment]
@mergifyio queue

**Comment by jacobweinstock** [comment]
@mergify queue

**Pull request merged**